Formula/hashcat.rb

```diff
@@ -60,7 +60,7 @@ def install
     # Test 1 (dictionary attack, -a 0):
     #
 
-    hash = File.open(hash_file, "rb").read.strip
+    hash = File.open(hash_file, "rb") { |f| f.read.strip }
```

Comment:
Previously it was not closing the file descriptor. Not a terrible problem here as it'll be closed when the entire process exits but if you do that in a loop: 💥. It's bad practise, regardless.